src/seismometer/controls/styles.py

```diff
@@ -3,9 +3,10 @@
 WIDE_LABEL_STYLE = {"description_width": "120px"}
 
 BOX_GRID_LAYOUT = Layout(
-    align_items="flex-start", grid_gap="20px", width="100%", min_width="300px", max_width="1400px"
+    align_items="flex-start", grid_gap="20px", width="100%", min_width="300px", max_width="1200px"
```

Comment:
Curious - why does this one move FROM 1400px to 1200px and everywhere else seems to move TO using 1400px?  
some kind of 200px buffer?

Reply:
moved everything to 1200px due to sphinx, this was an accidental exception.